pkg/generate/config/field.go

```diff
@@ -124,6 +124,9 @@ type PrintFieldConfig struct {
 	// include the field in `kubectl get` response. This field is generally used
 	// to override very long and redundant columns names.
 	Name string `json:"name"`
+	// Standard indicates whether the column/field should be shown in standard view or
+	// wide view (using the -o wide flag).
+	Standard bool `json:"standard"`
```

Comment:
This means that users should set `standard: true` in order to make the field printable with a simple `kubectl get`. Maybe i should rename it to `WideView` and inverse the default behavior ?

Reply:
How about calling the field `Priority` and making it an `*int`. If missing, default is `1` to align with the upstream kubebuilder behaviour.

Reply:
Done

Reply:
I just realized that default in kubebuilder is `0`. But i'm thinking that maybe keeping it to `1` is OK :thinking: